drivers/SmartThings/zigbee-switch/src/thirdreality/init.lua

```diff
@@ -0,0 +1,31 @@
+local capabilities = require "st.capabilities"
+
+local THIRDREALITY_MOTIOM_CLUSTER = 0xFC00
+local ATTR_TR = 0x0002
+local MOTION_DETECT = 0x0001
+local MOTION_NO_DETECT = 0x0000
+
+local function present_value_attr_handler(driver, device, value, zb_rx)
```

Comment:
rename this function to something more accurate like `motion_sensor_attr_handler` or `occupancy_attr_handler`